src/ReCaptcha/ReCaptcha.php

```diff
@@ -51,6 +51,12 @@ class ReCaptcha
      */
     const SITE_VERIFY_URL = 'https://www.google.com/recaptcha/api/siteverify';
 
+    /**
+     * User response token parameter name
+     * @const string
+     */
+    const USER_TOKEN_PARAMETER = 'g-recaptcha-response';
```

Comment:
Consider renaming to `USER_RESPONSE_TOKEN_PARAMETER` or `USER_RESPONSE_PARAMETER`. 

`USER_TOKEN_PARAMETER` does not ring a bell at first sight and the comment suggests better.